subscriptions.md

```diff
@@ -0,0 +1,297 @@
+        "id" : 1,
+        "name" : "Frequency",
+        "value" : "Daily"
+      } ],
+      "subscriptionType" : "TypeTest",
+      "_links" : {
+        "dSpaceObject" : {
+          "href" : "http://localhost/api/core/subscriptions/1/dSpaceObject"
+        },
+        "ePerson" : {
+          "href" : "http://localhost/api/core/subscriptions/1/ePerson"
+        },
+        "self" : {
+          "href" : "http://localhost/api/core/subscriptions/1"
+        }
+      }
+    } ]
+  },
+  "_links" : {
+    "self" : {
+      "href" : "http://localhost/api/core/subscriptions"
+    },
+    "search" : {
+      "href" : "http://localhost/api/core/subscriptions/search"
+    }
+  },
+  "page" : {
+    "size" : 20,
+    "totalElements" : 1,
+    "totalPages" : 1,
+    "number" : 0
+  }
+}
+```
+
+Attributes
+* page, size [see pagination](README.md#Pagination)
+
+Return codes:
+* 200 OK - if the operation succeed
+* 401 Unauthorized - if you are not authenticated. Please note that this also apply to resource policy related to the Anonymous group
+* 403 Forbidden - if you are not logged in with sufficient permissions. Only system administrators, users with ADMIN right can access the subscriptions
+
+## Single Subscription Object
+**GET /api/core/subscription/<:id>**
+
+Provide detailed information about a specific subscription.
+The JSON response document is as follow
+
+```
+{
+  "id" : 1,
+  "type" : "subscription",
+  "subscriptionParameterList" : [ {
+    "id" : 1,
+    "name" : "Frequency",
+    "value" : "Daily"
+  } ],
+  "subscriptionType" : "TestType",
+  "_links" : {
+    "dSpaceObject" : {
+      "href" : "http://localhost/api/core/subscriptions/1/dSpaceObject"
+    },
+    "ePerson" : {
+      "href" : "http://localhost/api/core/subscriptions/1/ePerson"
+    },
+    "self" : {
+      "href" : "http://localhost/api/core/subscriptions/1"
+    }
+  }
+}
+```
+
+Status codes:
+* 200 OK - if the subscription is found and it is visible to the current user.
+* 401 Unauthorized - if you are not authenticated and the subscription is not visible to anonymous users
+* 403 Forbidden - if you are not logged in with sufficient permissions. Only Admin and owner subscription has access.
+* 404 Not found - if the subscription doesn't exist
+
+### Search methods
+#### findByEPerson
+**GET /api/core/subscription/search/findByEPerson?uuid=<:ePerson-uuid>**
+
+The supported parameters are:
+* page, size [see pagination](README.md#Pagination)
+* uuid: mandatory, the uuid of the eperson that benefit of content subscription
+
+```json
+{
+  "_embedded": {
+    "subscriptions": [
+      {
+        "id": 60,
+        "type": "subscription",
+        "subscriptionParameterList" : [ {
+			    "id" : 2,
+			    "name" : "Frequency",
+			    "value" : "Daily"
+		  } ],
+        "subscriptionType": "subscription",
+        "_links": {
+          "dSpaceObject": {
+            "href": "http://localhost:8080/server/api/core/subscriptions/60/dSpaceObject"
```

Comment:
Tiny thing, but could we just call this "object" or "resource"?  I don't think we need to use the term "dspaceObject" in our REST API, as everything is obviously coming from DSpace.  In other endpoints, we tend to use "resource" or "object".

This same comment applies to everywhere you are using "dSpaceObject" in the REST API

Reply:
@tdonohue  thanks for the feedback, I updated it with 'resource' as found another endpoit[resourcepolicy.md] that in the same case uses param 'resource'.